KimYM/lab_1/checkout-b <KimYM_lab_1>

```diff
@@ -0,0 +1,48 @@
+#include <iostream>
+#include <conio.h>
+int main()
+{
+    char ch, mathaction;
+    double number1, number2;
+    setlocale(0, "");
+    do {
+        std::cout << "Введите первое число\n";
+        std::cin >> number1;
+        std::cout << "Введите второе число\n";
+        std::cin >> number2;
+        std::cout << "Введите знак математического действия\n";
+        std::cin >> mathaction;
+        if (mathaction == '+' || mathaction == '-' || mathaction == '*' || mathaction == '/') {
```

Comment:
You don't actually need to test mathaction for content as it's done later by switch..case